test/testdrive/apd-sum.td

```diff
@@ -232,28 +232,30 @@ $ file-append path=scaled_values.csv
 > CREATE MATERIALIZED VIEW apd_scaled_values_sum AS
     SELECT sum(a)::apd(39, 3) AS sum_a FROM apd_scaled_values;
 
+# Even though the sum is supposed to be rescaled to 3 decimals, the reudction
+# before packing the values into a row trims the trailing zeroes. However, the # rescale still works because the values are rounded to 3 units of scale.
```

Comment:
super minor but this comment looks like it got a bit misformatted; also "reudction"

Reply:
oh derp, ty--on a different PC today and fighting with vscode